### PR TITLE
New tasks in menu and rename files

### PR DESCRIPTION
**Pull request by skhomm**
Options 1, 2 are waiting for your pull requests.
And it's much easier now to discuss the project due to the new laconic filenames!

### CODE REVIEW

File: menu.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+This is the main module of the test suite.
+
+It runs submodules
+and possibly helps to configure parameters by going through menus.
+"""
+
+from make_roam.aruba import iap
+
+
+def menu():
```
Comment (ttl256) [inline menu.py:11]: This approach doesn't look scalable to me. Any new option would require changes in menu() and adding new elif below. Did you research other opportunities to implement interactive CLI menu?

**Review by ttl256 — COMMENTED** [review]
Please check my concern on menu() approach

File: menu.py (hunk 1 above)
Comment (skhomm) [reply menu.py:11]: Scale my ass! It's just a draft.
Would you be so kind and merge it to the master?

Comment (ttl256) [reply menu.py:11]: ¯\_(ツ)_/¯

**Comment by ttl256** [comment]
New menu and simplified filenames

**Pull request merged**